components/micro-gateway-interceptor/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>components</artifactId>
         <groupId>org.wso2.am.microgw</groupId>
-        <version>3.1.0</version>
+        <version>3.1.1</version>
```

Comment:
The version in the pom should be 3.1.1-SNAPSHOT, isn't it

Reply:
updated with https://github.com/wso2/product-microgateway/pull/2115/commits/0c3443f73140d824d65b34e0e252d723f72f894c